tests/testJointcal.py

```diff
@@ -75,8 +75,9 @@ def setUp(self):
         # position of the Twinkles run 1 catalog
         center = coord.IcrsCoord(53.00914*geom.degrees, -27.43895*geom.degrees)
         r = geom.Angle(3, geom.degrees)
-        astrometryTask = astrometry.AstrometryTask()
-        reference = astrometryTask.refObjLoader.loadSkyCircle(center, r, filterName='r')
+        config = astrom.LoadAstrometryNetObjectsTask.ConfigClass()
+        refLoader = astrom.LoadAstrometryNetObjectsTask(config=config)
```

Comment:
No, that's a lie: If I do `refLoader = LoadAstrometryNetObjectsTask()` I get an error that __init__() needs two 2 arguments.